<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting Simulation / 19124 Jeuk Hwang</string>
  </property>
  <widget class="QWidget" name="central">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
    <item>
     <widget class="PlotWidget" name="canvas_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_9" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="speed">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Speed : 10ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="array">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Size : 10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_size">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="sliderPosition">
           <number>10</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="jeuk">
        <property name="text">
         <string>2020-1 DS 19-124 Jeuk Hwang</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="shuffle">
          <property name="text">
           <string>Shuffle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="turn">
          <property name="text">
           <string>Turn :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time">
          <property name="text">
           <string>Time :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="sort_group">
        <property name="title">
         <string>Sorting algorithm</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="6" column="1">
             <widget class="QRadioButton" name="selection">
              <property name="text">
               <string>Selection</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QRadioButton" name="merge">
              <property name="text">
               <string>Merge</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QRadioButton" name="shell">
              <property name="text">
               <string>Shell</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QRadioButton" name="quick">
              <property name="text">
               <string>Quick</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QRadioButton" name="insertion">
              <property name="text">
               <string>Insertion</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="gnome">
              <property name="text">
               <string>Gnome</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="heap">
              <property name="text">
               <string>Heap</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="bubble">
              <property name="text">
               <string>Bubble</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="bogo">
              <property name="text">
               <string>Bogo</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="option_group">
        <property name="title">
         <string>Result Option</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QRadioButton" name="line_btn">
              <property name="text">
               <string>Line</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="point_btn">
              <property name="text">
               <string>Point</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="two_heatmap">
              <property name="text">
               <string>2D heatmap</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="three_heatmap">
              <property name="text">
               <string>3D heatmap</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
